CMakeLists.txt

```diff
@@ -9,5 +9,5 @@
 
 cmake_minimum_required( VERSION 3.3.2 FATAL_ERROR )
 
-project( ioda_data VERSION 2.0.0 DESCRIPTION "IODA Test Files" )
+project( ioda_data VERSION 2.2.0 DESCRIPTION "IODA Test Files" )
```

Comment:
why 2.2 and not 2.1?

Reply:
2.1 already existed beforehand, apparently this was forgotten last time round, too.